free-tunes/LogIn/LogInViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LogInViewModel.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/03/24.
+//
+
+import Foundation
+
+protocol LogInViewModelDelegate: AnyObject {
+    func notifyEmptyData()
+    func navigateToNext()
+    func show(error: String)
```

Comment:
Could this not be a more descriptive name than show?

Reply:
I believe it's more generic, hence I left it as is

Reply:
The function show will always return an error though as it takes in a parameter called error

Reply:
I believe it's descriptive enough